lib/blob/cluster/cluster.go

```diff
@@ -256,13 +261,13 @@ func (c *cluster) fetchObject(hash string) error {
 	for _, p := range peers {
 		objects, err := c.getObjects(p)
 		if err != nil {
-			c.Errorf("Failure to fetch %v from %v: %v.", hash, p, trace.DebugReport(err))
+			c.WithError(err).Errorf("Failure to fetch %v from %v.", hash, p)
```

Comment:
Nit: Not a big deal but I'd consider updating this and other logs to use fields for hash/peer.